src/please.go

```diff
@@ -169,6 +169,7 @@ var opts struct {
 		Parallel   struct {
 			NumTasks       int  `short:"n" long:"num_tasks" default:"10" description:"Maximum number of subtasks to run in parallel"`
 			Quiet          bool `short:"q" long:"quiet" description:"Suppress output from successful subprocesses."`
+			Output         run.ParallelOutput `short:"o" long:"output" description:"Output (default, quiet, group_immediate)" default:"default"`
```

Comment:
FYI you can get the flags library to handle some of this for you :)
```
choice:"default" choice:"quiet" choice:"group_immediate"
```
It should generate appropriate help text, but will also ensure the value is one of those choices.

Reply:
Nice!